### PR TITLE
#39 & #40 Keyboard Events - (Ticket Selection)

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

**Comment by mathvaillant** [comment]
@joel-gouveia the logging errors from jest are fixed in this PR.

**Comment by mathvaillant** [comment]
[Keyboard Events - Ticket types](https://trello.com/c/uH3pO7PF/39-keyboard-events-ticket-types)

**Comment by mathvaillant** [comment]
[Keyboard Events - Ticket Selection](https://trello.com/c/CEXRKgwU/40-keyboard-events-ticket-selection)

File: src/screens/TicketSelection/Item.tsx [hunk 1]
```diff
@@ -88,6 +88,10 @@ export function BookingItem({
   );
 }
 
+export const BookingItem = React.memo(BookingItemComponent, (prevProps, nextProps) => {
+  return prevProps.count === nextProps.count;
+});
```
Comment (joel-gouveia) [inline src/screens/TicketSelection/Item.tsx:93]: Do we need this? if only the count changes usually it will rerender. Is there a strong reason for memo?

Comment (mathvaillant) [reply src/screens/TicketSelection/Item.tsx]: Not really, just removed that!

**Review by joel-gouveia — APPROVED** [review]

**Pull request merged**